docs/refs.bib

```diff
@@ -304,6 +304,23 @@ @article{wilson2018maximizing
   year={2018}
 }
 
+@inproceedings{eriksson2019scalable,
+  title = {Scalable Global Optimization via Local {Bayesian} Optimization},
+  author = {Eriksson, David and Pearce, Michael and Gardner, Jacob and Turner, Ryan D and Poloczek, Matthias},
+  booktitle = {Advances in Neural Information Processing Systems},
+  pages = {5496--5507},
+  year = {2019},
+  url = {http://papers.nips.cc/paper/8788-scalable-global-optimization-via-local-bayesian-optimization.pdf},
+}
+
+@article{Moss2020BOSHBO,
```

Comment:
@henrymoss is this published? if so we should cite the conference / paper version, not arxiv

Reply:
this was a workshop paper, so I guess the answer is "no"?